modules/ossm-add-project-using-label-selectors-cli.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-adding-project-using-label-selectors-cli_{context}"]
+= Adding a project to the {SMProductShortName} using label selectors with the CLI 
+
+You can use labels selectors to add a project to the {SMProductShortName} with the CLI.
+
+.Prerequisites
+* An installed, verified {SMProductName} Operator.
+* An existing `ServiceMeshMemberRoll` resource.
+* To perform this task, a user must have mesh admin privileges.
+
+.Procedure
+
+. Log in to the {product-title} CLI.
+
+. Edit the `ServiceMeshMemberRoll` resource.
++
+[source,terminal]
+----
+$ oc edit smmr -n <controlplane-namespace>
+----
+
+. Modify the YAML file to include namespace label selectors in the `spec.memberSelectors` field of the `ServiceMeshMemberRoll` resource.
++
+[NOTE]
+====
+Instead of using the `matchLabels` field, you can also use the `matchExpressions` field in the selector.
```

Comment:
maybe we can add a reference to the kubernetes documentation about label selectors: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
WDYT?

Reply:
@fjglira generally speaking, we're discouraged from linking to the upstream docs. Historically, there was confusion about what from the upstream content we support downstream, and it created problems for Customer Support.